Makefile.js

```diff
@@ -135,14 +135,14 @@ function generateRulesIndex(basedir) {
 
     output += "    var rules = Object.create(null);\n";
 
-    find(basedir + "rules/").filter(fileType("js")).forEach(function(filename) {
+    find(`${basedir}rules/`).filter(fileType("js")).forEach(function(filename) {
         const basename = path.basename(filename, ".js");
 
-        output += "    rules[\"" + basename + "\"] = require(\"./rules/" + basename + "\");\n";
+        output += `    rules[\"${basename}\"] = require(\"./rules/${basename}\");\n`;
```

Comment:
Not that big of a deal I think, but a lot of the quotes in this diff no longer need to be escaped. However the output is still the same so not sure if worth the effort to clean them all up?